tdrs-frontend/src/utils/eventLogger.js

```diff
@@ -0,0 +1,43 @@
+import { useSelector } from 'react-redux'
+import axiosInstance from '../axios-instance'
+
+class EventLogger {
```

Comment:
Custom Logger class for our convenience. Note this can be extended with any generic `logEvents` function, in case we want to have more fine-tuned loggers in the future, or different endpoints, or if we end up using something like Sentry.

Reply:
I like this, very clean!